packages/SwingSet/src/kernel/slogger.js

```diff
@@ -230,8 +230,16 @@ export function makeSlogger(slogCallbacks, writeSlogObject, slogDuration) {
       deliveryNum = newDeliveryNum;
       replay = inReplay;
       syscallNum = 0;
-      const startProps = { crankNum, vatID, deliveryNum, replay, kd, vd };
-      const finish = startDuration(['deliver', 'deliver-result'], startProps);
+      const dispatch = vd?.[0]; // using vd because kd is undefined in replay
+      const finish = startDuration(['deliver', 'deliver-result'], {
+        crankNum,
+        vatID,
+        deliveryNum,
+        replay,
+        dispatch,
```

Comment:
Nit: consider keeping the order of the `replay` field consistent with syscall.

```suggestion
        dispatch,
        replay,
```